graphenex/core/web/__init__.py

```diff
@@ -25,6 +23,14 @@
 def run_server(args=None, exit_shell=True):
     """Run the web server"""
 
+    flask_secret_key = get_flask_secret_key()
+    if not flask_secret_key:
+        logger.error("`FLASK_SECRET_KEY` isn't set in `.env` file. Please configure it before running the Web UI.")
+        return
```

Comment:
From the UX perspective the following case is not very convenient:

- Run gX
- Run `web` command
- Get this error
- Go to the documentation to see how to set this
- Set the FLASK_SECRET_KEY
- Go back to step 1 and repeat

So maybe we can _prompt_ for a key in the terminal if it is not set. What do you think?

Reply:
From a UX perspective It could be a bit frustrating, but I don't really like getting the key from stdin for various reasons:

1. User will probably input a very weak key which isn't good from a security perspective
2. As we both know there is no (real) memory control in Python, so the password will linger in the heap for some time before getting cleaned up by the garbage collector

We could probably set a default strong secret key by default using `secrets`, but I'd much prefer if the user did that. It wouldn't take a considerable amount of time. LMK what you think about that.

Reply:
Good points. Can we at least add a CLI argument maybe? Something like `--web <token>` or `--flask-secret-key`?

Reply:
> Good points. Can we at least add a CLI argument maybe? Something like `--web <token>` or `--flask-secret-key`?

Yes, I'll add that - I've completely messed up the branch conflicts and don't know how to fix it. Let me open a new PR xD

Reply:
okay no worries :D